packages/node_modules/glimmer-runtime/lib/compiled/opcodes/dom.ts

```diff
@@ -178,7 +178,11 @@ export class NamespacedAttribute implements ElementPatchOperation {
   }
 
   apply(dom: DOMHelper, element: Element, lastValue: string = null): any {
-    let { value: reference, namespace } = this;
+    let {
+      name,
+      value: reference,
+      namespace
+    } = this;
     let value = reference.value();
```

Comment:
(Oops, this is one line off, I was again just surprised that the tests didn't catch the mistake!)